### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by kensteph**
In this PR We :
- Adding gitignore file
- Adding linter's configuration files.
- Add index.html file, book-list, and add-book sections
- Adding javascript file, adding functions addbook
- Add removeBook,displayBooks, and preserved the data in localStorage

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @kensteph & @alex1779, This is @sja-thedude your code reviewer.

While you made a great effort in this project, I cannot proceed to review your code.

You have submitted a project, that doesn't have a professional readme, make sure to add the updated one from the microverse template provided.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  }
+  section.innerHTML = books;
+}
+
+function removeBook(id) {
+  const bookId = parseInt(id, 2);
+  listBooks = listBooks.filter((book, index) => index !== bookId);
+  saveToLocalStorage();
+  displayBooks();
+}
```
Comment (AmaduKamara) [inline js/index.js]: You've done well so far in your implementation to delete a book, but there are a few changes to be made for the delete process to work fine as expected:

- [ ] Currently, when I add books and attempt to remove the last one or second to last, the book clicked cannot be removed. And also, the removal only happens for books at the top of the list. Kindly redo the deletion of a book so the right book is deleted whenever the remove button is clicked for a book.

- To achieve this, I recommend, when adding a book, the object you created that is pushed to the bookList array, you add the id property for a new book and give it a random id, so when you delete a book with the filter method, you compare the book clicked by the ID of the book filtered.

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used plain array and object for storing books 👍🏽
✅ Books are stored to localStorage correctly 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Reem-lab — APPROVED** [review]
# Hi @kensteph, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- No Linters error ✅
- Followed GitHub flow ✅
- Descriptive PR ✅
- Great working on the set up functionlaity ✅
- Professional README ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**